src/locales/__tests__/i18n-formatting.test.ts

```diff
@@ -38,7 +38,7 @@ test('formatting fiat currency with i18n', () => {
     }) as string
   })
 
-  expect(result.current).toBe('$123.46')
+  expect(result.current).toBe('$111,222,333,444,555,666,777,888,999.11')
```

Comment:
This one is failing

Reply:
Ah. Need to upgrade everything to Node 19

Reply:
(To match browser behaviour)